Traceback (most recent call last):
  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/codespace/.local/lib/python3.12/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/python/3.12.1/lib/python3.12/asyncio/base_events.py", line 684, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/codespace/.local/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/codespace/.local/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
ax = plt.subplots(figsize=(15,7))
ax = plt.plot(range(1,11),sse, linewidth=2, color="red", marker ="8")
plt.axvline(x=3, ls='--')
plt.ylabel('WCSS')
plt.xlabel('No. of Clusters (k)')
plt.title('The Elbow Method', fontsize = 20)
plt.show()
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[15], line 2[0m
[1;32m      1[0m ax [38;5;241m=[39m plt[38;5;241m.[39msubplots(figsize[38;5;241m=[39m([38;5;241m15[39m,[38;5;241m7[39m))
[0;32m----> 2[0m ax [38;5;241m=[39m plt[38;5;241m.[39mplot([38;5;28mrange[39m([38;5;241m1[39m,[38;5;241m11[39m),[43msse[49m, linewidth[38;5;241m=[39m[38;5;241m2[39m, color[38;5;241m=[39m[38;5;124m"[39m[38;5;124mred[39m[38;5;124m"[39m, marker [38;5;241m=[39m[38;5;124m"[39m[38;5;124m8[39m[38;5;124m"[39m)
[1;32m      3[0m plt[38;5;241m.[39maxvline(x[38;5;241m=[39m[38;5;241m3[39m, ls[38;5;241m=[39m[38;5;124m'[39m[38;5;124m--[39m[38;5;124m'[39m)
[1;32m      4[0m plt[38;5;241m.[39mylabel([38;5;124m'[39m[38;5;124mWCSS[39m[38;5;124m'[39m)

[0;31mNameError[0m: name 'sse' is not defined

